.scalafmt.conf

```diff
@@ -1,21 +1,52 @@
-version = 2.1.0
-
-style = defaultWithAlign
-
-docstrings                 = JavaDoc
-indentOperator             = spray
-maxColumn                  = 120
-lineEndings                = preserve
-rewrite.rules              = [RedundantParens, SortImports, AvoidInfix]
-unindentTopLevelOperators  = true
-align.tokens               = [{code = "=>", owner = "Case"}]
-align.openParenDefnSite    = false
-align.openParenCallSite    = false
-optIn.breakChainOnFirstMethodDot = false
-optIn.configStyleArguments = false
-danglingParentheses = false
-spaces.inImportCurlyBraces = true
-rewrite.neverInfix.excludeFilters = [
+version                                  = 3.6.1
+runner.dialect                           = scala213
+project.git                              = true
+style                                    = defaultWithAlign
```

Comment:
I'm not very fond of vertical alignment, because it generates whitespace diffs, and because code can get awkward when the length of the values / identifiers differs.

It's safer if we don't do vertical alignment. Although, personally, I can live with it.

Reply:
> I'm not very fond of vertical alignment, because it generates whitespace diffs, and because code can get awkward when the length of the values / identifiers differs. We align <- in for-comprehensions at $work, and I often find myself renaming identifiers in order for the for expression to look good (that, or starting code blocks, to make the declarations multi-line).

> It's safer if we don't do vertical alignment. Although, personally, I can live with it.

So I am going to remove the `!` vertical alignment because it visually doesn't look well, but I would opt for leaving operators such as `~>` since it does greatly improve clarity (`~>` is used in streams with graphs and since we are dealing with graphs the alignment helps in seeing the port flow).

The point about whitespace and diff is a valid one however github now supports viewing diffs while ignoring whitespace which does help immensely https://github.blog/2018-05-01-ignore-white-space-in-code-review/.

Also note the other operators are there because the old scalafmt also had them so the goal here is to minimize the diff.

Reply:
> Also note the other operators are there because the old scalafmt also had them so the goal here is to minimize the diff.

Good argument 👍

Reply:
Cool I am going to resolve this, feel free to reopen + bump if there is something else.